util/reward-calculator/src/lib.rs

```diff
@@ -147,7 +147,7 @@ impl<'a, CS: ChainStore<'a>> RewardCalculator<'a, CS> {
         let mut proposed: HashSet<ProposalShortId> = HashSet::default();
         let mut index = parent.to_owned();
 
-        let committed_idx_proc = |hash: &H256| -> HashSet<ProposalShortId> {
+        let committed_idx_proc = |hash: &H256| -> Vec<ProposalShortId> {
```

Comment:
It's better to add a comment that it must return ids in the same order as the fees returned from `txs_fees_proc`.